From 65667160ed5f1ae2dc3b9f88d237e9f5a3ba3bcd Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@denx.de>
Date: Sun, 11 Jun 2023 22:22:25 +0200
Subject: [PATCH 1/2] kmssink: Add NXP i.MX8M Plus LCDIFv3 auto-detection

Add i.MX8M Plus LCDIFv3 controller into list of auto-detected modules.

Upstream-Status: Pending
Signed-off-by: Marek Vasut <marex@denx.de>
---
 sys/kms/gstkmssink.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sys/kms/gstkmssink.c b/sys/kms/gstkmssink.c
index db983439e9..923e729861 100644
--- a/sys/kms/gstkmssink.c
+++ b/sys/kms/gstkmssink.c
@@ -174,8 +174,8 @@ static int
 kms_open (gchar ** driver)
 {
   static const char *drivers[] = { "i915", "radeon", "nouveau", "vmwgfx",
-    "exynos", "amdgpu", "imx-drm", "rockchip", "atmel-hlcdc", "msm",
-    "xlnx", "vc4", "meson", "sun4i-drm", "mxsfb-drm",
+    "exynos", "amdgpu", "imx-drm", "imx-lcdif", "rockchip", "atmel-hlcdc",
+    "msm", "xlnx", "vc4", "meson", "sun4i-drm", "mxsfb-drm",
     "xilinx_drm",               /* DEPRECATED. Replaced by xlnx */
   };
   int i, fd = -1;
-- 
2.39.2

